README.md

```diff
@@ -22,7 +22,7 @@ For example:
 import {
   abi as SWAP_ROUTER_ABI,
   bytecode as SWAP_ROUTER_BYTECODE,
-} from '@uniswap/swap-router-contracts/artifacts/contracts/SwapRouter02.sol/SwapRouter02.json'
+} from '@uniswap/swap-router-contracts/artifacts/contracts/ApeSwapMultiSwapRouter.sol/ApeSwapMultiSwapRouter.json'
```

Comment:
Would these contracts come from `@ape.swap` instead? (Same for the one below?)

Reply:
If we want a package for this? But don't see the need. This was just a renameAll